lib/rules/require-padding-newlines-before-line-comments.js

```diff
@@ -52,8 +52,20 @@ module.exports.prototype = {
                 return;
             }
 
+            var prevToken = file.getPrevToken(comment, {
+                includeComments: true
+            });
+
+            if (prevToken.type === 'Line') {
+                return;
+            }
+
+            if (prevToken.type === 'Punctuator' && prevToken.value === '{') {
```

Comment:
There are #698 and #136 (probably something else) where we can change the default value. And also #1064 could help us to know what default value should be for each rule. ;-)